processor/transformprocessor/README.md

```diff
@@ -786,9 +880,7 @@ The feature is currently only available for log processing.
   transform:
     flatten_data: true
     log_statements:
-      - context: log
-        statements:
-          - set(resource.attributes["to"], attributes["from"])
+      - set(log.resource.attributes["to"], log.attributes["from"])
```

Comment:
```suggestion
      - set(resource.attributes["to"], log.attributes["from"])
```
This should just be `resource.attributes`, right?

Reply:
Good catch! Thanks!